i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/quick_deploy.md

```diff
@@ -214,7 +214,7 @@ export ENABLE_SPARK=false
 
 install.sh脚本会询问您是否需要初始化数据库并导入元数据。如果选择初始化，会把数据库中的表数据清空重新初始化。
 
-<font size="6" color="red">第一次安装</font><font color="red">必须选清空数据库</font>
+**<font color="red">第一次安装必须选清空数据库</font>**
```

Comment:
Modify the corresponding documents of other versions together

Reply:
added